nixos/modules/services/networking/shadowsocks.nix

```diff
@@ -129,7 +147,7 @@ in
       description = "shadowsocks-libev Daemon";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      path = [ pkgs.shadowsocks-libev cfg.plugin ] ++ optional (cfg.passwordFile != null) pkgs.jq;
+      path = [ pkgs.shadowsocks-libev ] ++ optional (cfg.plugin != null) cfg.plugin ++ optional (cfg.passwordFile != null) pkgs.jq;
```

Comment:
This should fix https://github.com/NixOS/nixpkgs/pull/96127#issuecomment-688849348

Reply:
LGTM :+1: